index.html

```diff
@@ -266,31 +302,36 @@ <h3 class="skill-img"> <img src="img/round57.png" alt=""> Skill</h3>
     </div>
    </section>
    <!--contact section start-->
- <section class="contact-form-container">
-    <p>
-        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-    </p>
-    <form action="https://formspree.io/f/xdovandv" method="post">
-        <ul class="form-input">
-            <li>
-                <label for="name">
-                <input type="text" id="name" name="name" maxlength="30" required placeholder="Full name">
-            </label>
-            </li>
-            <li>
-                <label for="email">
-                <input type="email" id="email" name="email" placeholder="Email address" required>
-            </label>
-            </li>
-            <li>
-                <label for="message">
-                <textarea type="text" id="message" name="message" maxlength="500" required placeholder="Enter text here"></textarea></label>
-            </li>
-            <li>
-                <button type="submit">GET IN TOUCH</button>
-            </li>
-        </ul>
-     </form>
+ <section class="contact-me">
```

Comment:
- [ ] Kindly align the content of the contact me section as a row and not as a column to match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215283516-600e2128-59e1-4f67-821c-5a877a222a85.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215283528-29356b5b-e450-4b1f-a610-a205f9885f15.png)